sdk/endpoints/online/managed/online-endpoints-managed-identity-sai.ipynb

```diff
@@ -0,0 +1,473 @@
+   "outputs": [],
+   "source": [
+    "from azure.identity import DefaultAzureCredential, AzureCliCredential \n",
+    "from azure.mgmt.storage import StorageManagementClient\n",
+    "from azure.storage.blob import ContainerClient\n",
+    "from azure.mgmt.storage.models import Sku, StorageAccountCreateParameters, BlobContainer"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "name" : "create-storage-2"
+   },
+   "outputs": [],
+   "source": [
+    "credential = AzureCliCredential()\n",
+    "storage_client = StorageManagementClient(credential, subscription_id,)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "name" : "create-storage-3"
+   },
+   "outputs": [],
+   "source": [
+    "storage_account_parameters = StorageAccountCreateParameters(\n",
+    "    sku=Sku(name=\"Standard_LRS\"), \n",
+    "    kind=\"Storage\", \n",
+    "    location=location\n",
+    ")\n",
+    "\n",
+    "storage_client.storage_accounts.begin_create(\n",
+    "    resource_group_name=resource_group,\n",
+    "    account_name=storage_account_name, \n",
+    "    parameters=storage_account_parameters)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "name" : "create-storage-4"
+   },
+   "outputs": [],
+   "source": [
+    "storage_account = next((\n",
+    "    account for account in \n",
+    "    storage_client.storage_accounts.list_by_resource_group(resource_group)\n",
+    "    if account.name==storage_account_name\n",
+    "))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "name" : "create-storage-5"
+   },
+   "outputs": [],
+   "source": [
+    "blob_container = storage_client.blob_containers.create(\n",
+    "    resource_group_name=resource_group, \n",
+    "    account_name=storage_account_name,\n",
+    "    container_name=storage_container_name,\n",
+    "    blob_container=BlobContainer()\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "name" : "create-storage-6"
+   },
+   "outputs": [],
+   "source": [
+    "res = storage_client.storage_accounts.list_keys(\n",
+    "    resource_group_name=resource_group, \n",
+    "    account_name=storage_account_name,\n",
+    ")\n",
+    "key = res.keys[0].value"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "name" : "create-storage-7"
+   },
+   "outputs": [],
+   "source": [
+    "container_client = ContainerClient(\n",
+    "    account_url=storage_account.primary_endpoints.blob, \n",
+    "    container_name=storage_container_name,\n",
+    "    credential=key\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "name" : "create-storage-8"
+   },
+   "outputs": [],
+   "source": [
+    "file_path = \"endpoints/online/managed/managed-identities/hello.txt\" \n",
```

Comment:
Could you place `hello.txt` in the same directory and add in the PR?
(You can just copy the file from `cli/endpoints/online/managed/managed-identities/hello.txt`)

Reply:
Added